README.md

```diff
@@ -1,24 +1,36 @@
-# README
+# Micro Reddit
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A simple Ruby on Rails App - Building with Active Record.
 
-Things you may want to cover:
 
-* Ruby version
+Simple app built for practice, working with rails models and validations.
```

Comment:
Try to explain the application more extensively, it's a good way of understanding the concepts better and the readme file would look cleared to visitors.

Reply:
Explained the project in more detail as you proposed.